src/datreant/core/tests/test_collections.py

```diff
@@ -412,6 +412,137 @@ def test_map(self, collection, tmpdir):
         assert collection.map(return_nothing) is None
         assert collection.map(return_nothing, processes=2) is None
 
+    def test_flatten(self, collection, tmpdir):
```

Comment:
Not sure where this came from... was this removed and I've accidentally reintroduced it?